lit/docs/guides/git.lit

```diff
@@ -0,0 +1,9 @@
+\title{Git Guides}
+
+\use-plugin{concourse-docs}
+\split-sections
+\table-of-contents
+
+\section{
```

Comment:
Are you working on the `Git Guides` content within this PR? Or if we want to write the content for it in a later PR, we could remove this section for now and then add it back later?

Reply:
I have a one that I want to add as part of this PR. Wanted to submit what I had so far to get feedback on the direction 😀